x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/sequence/CircuitBreakerTests.java

```diff
@@ -308,6 +223,107 @@ private void assertMemoryCleared(int sequenceFiltersCount, BiFunction<CircuitBre
         }
     }
 
+    public void testParentCircuitBreakerOnClean() {
+        int sequenceFiltersCount = 2;
+        final int SEARCH_REQUESTS_EXPECTED_COUNT = 2;
```

Comment:
Nit: I guess this is c&p'd and that's why it's final/capitalized, but it looks a bit off that, comparatively, the var above is only effectively final and not actually so.